### PR TITLE
feat(azure): Add support for Azure Load Balancer 

### PR DESCRIPTION
**Pull request by NeverovDS**
Issue #666

### CODE REVIEW

**Comment by NeverovDS** [comment]
@ekifel ready for review

File: internal/providers/terraform/azure/lb_outbound_rule.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func GetAzureRMLoadBalancerOutboundRuleRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_lb_outbound_rule",
+		RFunc: NewAzureRMLoadBalancerRule,
+		ReferenceAttributes: []string{
+			"resource_group_name",
```
Comment (ekifel) [inline internal/providers/terraform/azure/lb_outbound_rule.go:14]: I think we should add `loadbalancer_id` here and in `lb_rule` resource, as @alikhajeh1 said in details

**Review by ekifel — APPROVED** [review]
Thanks @NeverovDS! One minor change left

**Comment by NeverovDS** [comment]
@ekifel check plz

**Pull request merged**